lms/djangoapps/discussion_api/api.py

```diff
@@ -391,7 +391,10 @@ def get_comment_list(request, thread_id, endorsed, page, page_size, mark_as_read
     # existing comments service interface
```

Comment:
Did you verify why this comment exists here?  This PR goes against the original design of this implementation.  We need to update this comment and also the docstring above: 

```
endorsed: Boolean indicating whether to get endorsed or non-endorsed
           comments (or None for all comments). Must be None for a discussion
           thread and non-None for a question thread.
```